openeo/rest/datacube.py

```diff
@@ -318,6 +318,17 @@ def filter_bbox(
                 >>> west, east, north, south = 3, 4, 52, 51
                 >>> cube.filter_bbox(west, east, north, south)
 
+        :param crs: data structure that encodes a CRS, typically just an int or string value.
+            If the ``pyproj`` library is available, everything supported by it is allowed.
```

Comment:
I would put this note about pyproj at the bottom to make it more clear what is minimally supported and what is supported when pyproj is available.

Also that "everything" sounds quite wide, I would make it a bit more explicit: "everything supported by `pyproj.CRS.from_user_input` is allowed", maybe with a link to https://pyproj4.github.io/pyproj/dev/api/crs/crs.html#pyproj.crs.CRS.from_user_input  (not sure how stable that link is)

Reply:
Done:
Removed comment about pyproj in this docstring, just referring to  :py:func:`openeo.util.normalize_crs.
And clarified "everything supported by ..." in normalize_crs with link to corresponding pyproj docs page.